### PR TITLE
Make wallet able to send to multiple destinations

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: BTCPayServer/Controllers/WalletsController.cs [hunk 1]
```diff
@@ -215,17 +222,77 @@ public async Task<IActionResult> ListWallets()
...
+                var index = int.Parse(command.Substring(command.IndexOf(":",StringComparison.InvariantCultureIgnoreCase) + 1),  CultureInfo.InvariantCulture);
+                vm.Outputs.RemoveAt(index);
+                return View(vm);
+            }
+            
+
+            if (!vm.Outputs.Any())
+            {
+                ModelState.AddModelError(string.Empty,
+                    "Please add at least one transaction output");
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/WalletsController.cs]: you can early return here and remove the `else`.

File: BTCPayServer/Controllers/WalletsController.cs [hunk 2]
```diff
@@ -215,17 +222,77 @@ public async Task<IActionResult> ListWallets()
...
+            }
+            else
+            {
+                var subtractFeesOutputsCount = new List<int>();
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/WalletsController.cs]: Should not use checkboxes but radio boxes! (Like I did in WalletSettings)

**Comment by NicolasDorier** [comment]
On top of my comments. I think the Add destination should be next to the `Sign` button.
Also, if there is no multiple destination, the UI should be the same as before. 

That way, it does not scare noobs and you don't have to change the selenium tests.

File: BTCPayServer/Controllers/WalletsController.cs (hunk 2 above)
Comment (NicolasDorier) [reply BTCPayServer/Controllers/WalletsController.cs]: Actually we can't use radio box I think. Radio box, at least one much be checked right?

Comment (Kukks) [reply BTCPayServer/Controllers/WalletsController.cs]: Yup, that's why I didn't use a radiobox

**Comment by Kukks** [comment]
Done, the feature is very unintrusive :) 

image
image
image

**Comment by NicolasDorier** [comment]
Can we even remove the group box if there is only one destination?

**Comment by Kukks** [comment]
image
done

**Pull request merged**